app/src/main/java/com/example/series_collector/data/source/YoutubeDataSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.series_collector.data.source
+
+import com.example.series_collector.data.api.YouTubeService
+import javax.inject.Inject
+
+class YoutubeDataSource  @Inject constructor(
```

Comment:
```suggestion
class YouTubeDataSource  @Inject constructor(
```
사소하지만 youtube, youTube 네이밍이 섞여있는듯한데 하나로 통일해주시는게 좋을듯합니다 :)